.devcontainer/devcontainer.json

```diff
@@ -13,7 +13,7 @@
 		"PATH": "${containerEnv:PATH}:${containerEnv:FLUTTER_ROOT}/bin"
 	},
 	"onCreateCommand": "git clone --branch=stable https://github.com/flutter/flutter.git ${containerEnv:FLUTTER_ROOT} && ${containerEnv:FLUTTER_ROOT}/bin/flutter doctor",
-	"postCreateCommand": "git submodule update --init --recursive && flutter pub get && pnpm install",
+	"postCreateCommand": "git submodule update --init --recursive && flutter pub get && corepack install && pnpm install",
```

Comment:
Going from https://www.npmjs.com/package/corepack I'm guessing this is meant to be `corepack enable`

Reply:
Is that true, @jarrodcolburn? Should it be `corepack enable`, instead of `install`?

Reply:
Here's my belief...
* `corepack` is already enabled in the container 
* `corepack install` needs to run in the project's root